tests/foreman/cli/test_computeresource_azurerm.py

```diff
@@ -127,7 +135,16 @@ def test_positive_crud_azurerm_cr(self, module_org, module_location, azurerm_set
 
     @upgrade
     @tier2
-    @parametrize("image", [AZURERM_RHEL7_FT_IMG_URN, AZURERM_RHEL7_UD_IMG_URN])
+    @parametrize(
+        "image",
+        [
+            AZURERM_RHEL7_FT_IMG_URN,
+            AZURERM_RHEL7_UD_IMG_URN,
+            AZURERM_RHEL7_FT_BYOS_IMG_URN,
+            AZURERM_RHEL7_FT_CUSTOM_IMG_URN,
+            AZURERM_RHEL7_FT_GALLERY_IMG_URN,
+        ],
```

Comment:
added and updated as required to test all combinations or images.

Reply:
Also, dont forget to attach pytest ids with this parameterization else the collection will fail. For examples look at `foreman/tests/api/test_contentview.py`